app/src/main/java/io/github/droidkaigi/confsched2017/service/DebugOverlayService.java

```diff
@@ -0,0 +1,84 @@
+package io.github.droidkaigi.confsched2017.service;
+
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Handler;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched2017.service.helper.OverlayViewManager;
+import io.github.droidkaigi.confsched2017.util.SettingsUtil;
+import io.github.droidkaigi.confsched2017.view.activity.MainActivity;
+import timber.log.Timber;
+
+/**
+ * Created by KeishinYokomaku on 2017/02/12.
+ */
+
+public class DebugOverlayService extends BaseService {
+    private final BroadcastReceiver configurationChangeReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            handler.post(() -> {
+                Timber.d("onConfigurationChange");
```

Comment:
Actually Timber does that for us without calling `.tag(TAG)` explicitly so unless some tag customization is needed we don't need to call it.